content/docs/utilities/z-index.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Z-Index
+skellyCSS: true
+---
+
+{{% anchor name="Z-Index" %}}
+
+You can use these utility classes to control the stack. Add a specified `z-{value}` class to your element to change the order of the stack. Remember to specify at least `position: relative` on the element for `z-index` to work.
+
+{{< classes result="true" >}}
+{{< classes-row class="z-0" result="z-index: 0" >}}
```

Comment:
I'd put these results so they spit out `z-index: 0` instead of z-index: 0

Reply:
Oh you mean put the little code ticks around them?

Reply:
<img width="955" alt="Screen Shot 2022-08-02 at 11 50 36 AM" src="https://user-images.githubusercontent.com/5313708/182417703-5b06f7cc-de1f-4580-b2e0-ad75f8aca809.png">